tox.ini

```
@@ -41,6 +41,10 @@ commands =
     lint: yamllint -s test/ molecule/
 whitelist_externals =
     find
+# sitepackages=true <- template module <- libselinux bindings which cannot be
+# be installed with pip. Also this lowers virtualenvs footprints and install
+# time
+sitepackages=true
```

wrap it with whitespaces to match the common style 
```suggestion
sitepackages = true
```